<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SlicerSynthSeg</class>
 <widget class="qMRMLWidget" name="SlicerSynthSeg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   
   <!-- Configuration Section -->
   <item>
    <widget class="ctkCollapsibleButton" name="configurationCollapsibleButton">
     <property name="text">
      <string>Configuration</string>
     </property>
     <property name="collapsed">
      <bool>false</bool>
     </property>
     <layout class="QFormLayout" name="formLayout_config">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="configStatusLabel">
        <property name="text">
         <string>âš  Not configured</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="configureButton">
        <property name="text">
         <string>Configure Environment</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="testConfigButton">
        <property name="text">
         <string>Test Configuration</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>

   <!-- Parameters Section -->
   <item>
    <widget class="ctkCollapsibleButton" name="parametersCollapsibleButton">
     <property name="text">
      <string>Parameters</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input Volume:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="qMRMLNodeComboBox" name="inputVolumeSelector">
        <property name="nodeTypes">
         <stringlist>
          <string>vtkMRMLScalarVolumeNode</string>
         </stringlist>
        </property>
        <property name="showChildNodeTypes">
         <bool>false</bool>
        </property>
        <property name="addEnabled">
         <bool>false</bool>
        </property>
        <property name="removeEnabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>

      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="applyButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Run Segmentation</string>
        </property>
       </widget>
      </item>

     </layout>
    </widget>
   </item>

   <!-- Spacer -->
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>

  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
